src/main/resources/assets/js/application/interviewer/services/tdprPersonService.js

```diff
@@ -0,0 +1,12 @@
+define(['angular', 'application/interviewer/tdprInterviewerModule'], function(angular, tdprInterviewerModule) {
+    tdprInterviewerModule.service("tdprPersonService", function($http) {
+
+        this.getNote = function(personId, date) {
+            return $http.get("/api/person/getNote?" + "id=" + personId + "&date=" + date);
+        };
+
+        this.updateNote = function(note) {
+            return $http.put("/api/person/updateNote", note);
```

Comment:
Remember that we don't need to name the endpoint getNote, updateNote - the http methods define the actions, so a get to /api/person/1/note?date=today will get the note for person1 for today. A PUT call to /api/person/1/note will update note. naming endpoints like this is redundant